### PR TITLE
Release v2.0.0-rc.1

### PR DESCRIPTION
**Pull request by Udumft**

### CODE REVIEW

**Comment by S2Ler** [comment]
@Udumft would be nice to apply labels as in previous release. Similar is done in other repositories:

https://github.com/mapbox/mapbox-navigation-ios/releases/tag/v2.0.0-beta.25
https://github.com/mapbox/mapbox-navigation-ios/releases/tag/v2.0.0-beta.24
https://github.com/mapbox/mapbox-navigation-ios/releases/tag/v2.0.0-beta.23

⚠️ - breaking change
🐞 - fixed bug
🎁 - new feature
🔄 - an update to existing functionality

**Comment by S2Ler** [comment]
I will add `Known issues` about billing

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ### Pricing
 
-* New per trip pricing option. For more information, see the “[Pricing](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/)” guide ([#3147](https://github.com/mapbox/mapbox-navigation-ios/pull/3147))
+* New per trip pricing option. For more information, see the “[Pricing](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/)” guide ([#3147](https://github.com/mapbox/mapbox-navigation-ios/pull/3147), [#3338](https://github.com/mapbox/mapbox-navigation-ios/pull/3338))
 * `MBXNavigationBillingMethod` is no longer supported. ([#3147](https://github.com/mapbox/mapbox-navigation-ios/pull/3147))
 
 ### Packaging
```
Comment (chezzdev) [inline CHANGELOG.md:10]: It's interesting why don't we mention `MapboxSpeech` as a dependency in the changelog? Should we start doing it now?

Comment (S2Ler) [reply CHANGELOG.md:10]: Probably not. @1ec5?

**Review by S2Ler — APPROVED** [review]
🚀

**Pull request merged**